go-controller/pkg/ovn/ovn.go

```diff
@@ -302,6 +308,8 @@ func NewOvnController(ovnClient *util.OVNClientset, wf *factory.WatchFactory, st
 		egressSvcController:       egressSvcController,
 		podRecorder:               metrics.NewPodRecorder(),
 	}
+	atomic.StoreUint32(&ovnController.allInitialPodsProcessed, 0)
```

Comment:
@JacobTanenbaum actually we dont' need a store here since nothing else will be accessing it yet... can just leave it out entirely and the compiler will init to 0.